packages/plc/src/server/db.ts

```diff
@@ -1,44 +1,61 @@
+import { Kysely, SqliteDialect } from 'kysely'
+import SqliteDB from 'better-sqlite3'
 import { CID } from 'multiformats/cid'
 import { cidForData } from '@adxp/common'
-import {
-  EntityManager,
-  DataSource,
-  Entity,
-  Column,
-  PrimaryColumn,
-  In,
-  Not,
-} from 'typeorm'
-import { Mutex } from 'async-mutex'
 import * as document from '../lib/document'
 import * as t from '../lib/types'
 import { ServerError } from './error'
 
-export class Database {
-  db: SafeDB
+interface OperationsTable {
+  did: string
+  operation: string
+  cid: string
+  nullified: 0 | 1
```

Comment:
If we want to wrangle this quirk using types, we may be able to use an enum.  E.g. something along the lines of:
```ts
enum DbBool {
  true = 1,
  false = 0,
}
```

Reply:
im not too worried about it just yet. if we're feeling friction from it in the future, then totally 👌